assets/js/init.js

```diff
@@ -104,6 +106,17 @@ const x = {
           ticking = true;
         }
       });
+
+      // Toggle between display none and block for mobile navbar
+      toggler.addEventListener('click', () => { 
+        if(nav.className == "nav"){
```

Comment:
This mechanism seems rocky, add and remove a `nav-small` class *on top of* the existing `nav` class to prevent duplication of CSS